scripts/nym-node-setup/auto-bond/auto_bond_all.py

```
@@ -24,32 +28,30 @@
 SENSITIVE_FLAGS = {"--mnemonic", "--signature"}
 
 
 def ok(msg):   print(f"  {G}✓{NC} {msg}")
 def err(msg):  print(f"  {R}✗{NC} {msg}")
 def info(msg): print(f"  {C}→{NC} {msg}")
-def dim(msg):  print(f"  {D}{msg}{NC}")
+
+
+def nym_to_unym(nym) -> str:
+    """NYM (str/float) -> integer unym as a string for the CLI."""
+    return str(int(round(float(nym) * UNYM_PER_NYM)))
```

Ruff (0.16.3)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)